### ISSUE TITLE
[IMRPOVEMENTS]: more building increments 

### ISSUE THREAD

**Issue by JackOfCool** [typical-developers/oaklands-feedback#12]
### Roblox Username

JackThePC_Overlord

### What do you think can have quality-of-life improvements?

Add more building increments 

### How do you think things can be improved?

Add more increments to move and rotate
For example:
Instead of .25 .5 1 
It would be like 
.25 .5 .75 1 1.25 1.5 1.75 2 ... And so on 

I would make it so you can just type in any number (rounding to the nearest increment) 

It would be the same for rotation, so like 11.25, 22.5 33.75, 45 ect 

This would be useful for building, especially for percise structures, as well as just saving time in certain cases

**Comment by ManancialGD** [comment]
Yes! And I think it would be nice to choose the material of the base
